L1Trigger/TrackTrigger/python/ProducerHPH_cfi.py

```diff
@@ -1,11 +1,15 @@
 import FWCore.ParameterSet.Config as cms
+from L1Trigger.TrackFindingTMTT.TMTrackProducer_Defaults_cfi import TMTrackProducer_params
 
 # ParameterSet used by HitPatternHelper
 
 HitPatternHelper_params = cms.PSet (
 
   hphDebug   = cms.bool(False),   
   useNewKF   = cms.bool(False),
-  deltaTanL  = cms.double(0.125)
-
+  oldKFPSet  = cms.PSet(TMTrackProducer_params.EtaSectors),
+  deltaTanL  = cms.double(0.125),
```

Comment:
Add comments explaining what these 3 parameters are and how the numbers are chosen. Also, I think deltaTanL is only used in your code if the NewKF is being used?

Reply:
Done